agenta-cli/agenta/sdk/decorators/llm_entrypoint.py

```diff
@@ -79,6 +79,15 @@ async def wrapper(*args, **kwargs) -> Any:
             func_params, api_config_params = self.split_kwargs(kwargs, config_params)
             self.ingest_files(func_params, ingestible_files)
             agenta.config.set(**api_config_params)
+
+            # Set the configuration and environment of the LLM app parent span at run-time
+            agenta.tracing.set_span_attribute(
+                attributes={"config": config_params}, is_parent_span=True
```

Comment:
Why do we need to specify is_parent_span here? Doesn't the tracing class know whether the active span is a parent_span or not?